src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public String getRandomBall() {
+        String color = getColorSupplier().getRandomColor();
+        Long number = getRandom().nextLong(100);
+        return new Ball(color, number).toString();
```

Comment:
The `getRandomBall` method uses a `String` for the color when creating a `Ball` object. Since the `Ball` class expects a `String` for color, this is technically correct. However, if the `ColorSupplier` is updated to return a `Color` type instead of a `String`, you should adjust the `Ball` constructor to accept a `Color` type for better type safety.

Reply:
It's done, now you can merge